SonicWall/sonicwall-fw/tests/test_filename_logging.json

```diff
@@ -0,0 +1,100 @@
+{
+  "input": {
+    "sekoiaio": {
+      "intake": {
+        "dialect": "SonicWall Firewall",
+        "dialect_uuid": "ee0b3023-524c-40f6-baf5-b69c7b679887"
+      }
+    },
+    "message": "CEF:0|SonicWall|NSa 4500|7.0.1-1234-R5678|1574|Filename Logging|5|cat=0 gcat=3 smac=00:11:22:33:44:55 src=10.0.30.40 spt=12345 deviceInboundInterface=X3-V333 cs3Label=WORKSTATION dmac=66:77:88:99:00:11 dst=22.3.4.55 dpt=444 deviceOutboundInterface=X3-V33 cs4Label=LAN susr=\"USER\" proto=tcp/445 out=32701234 in=31445678 cs5Label=\"Auto\" cs1=\"555 (WORKSTATION->WORK)\" app=9876 msg=\"Filename: FILENAME\" cnt=123456789 fw_action=\"NA\" dpi=1"
+  },
+  "expected": {
+    "message": "CEF:0|SonicWall|NSa 4500|7.0.1-1234-R5678|1574|Filename Logging|5|cat=0 gcat=3 smac=00:11:22:33:44:55 src=10.0.30.40 spt=12345 deviceInboundInterface=X3-V333 cs3Label=WORKSTATION dmac=66:77:88:99:00:11 dst=22.3.4.55 dpt=444 deviceOutboundInterface=X3-V33 cs4Label=LAN susr=\"USER\" proto=tcp/445 out=32701234 in=31445678 cs5Label=\"Auto\" cs1=\"555 (WORKSTATION->WORK)\" app=9876 msg=\"Filename: FILENAME\" cnt=123456789 fw_action=\"NA\" dpi=1",
```

Comment:
Same as above: we can extract the filename from the message and add it to `file.name`